repo_create/test-1.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/python -tt
+
+from rpmfluff import SimpleRpmBuild
+from rpmfluff import YumRepoBuild
+import os
+import shutil
+
+work_file = os.path.realpath(__file__)
+work_dir = os.path.dirname(work_file)
+file_base_mane = work_file.rsplit('/', 1)[1].rsplit('.', 1)[0]
```

Comment:
if we use Python3, then take advantage of [pathlib](https://docs.python.org/3/library/pathlib.html)